### ISSUE TITLE
Libc not available when build c project targets "arrch64-linux-android" 

### ISSUE THREAD

**Issue by isJDongYa** [ziglang/zig#22163]
### Zig Version

0.13.0

### Steps to Reproduce and Observed Behavior

I build Flecs.Net.Native for "arrch64-linux-android", and it emit "Libc not available".

More msg see [https://github.com/BeanCheeseBurrito/Flecs.NET/discussions/78](url)

### Expected Behavior

Successfully build android lib  "libflecs.so"

**Comment by alexrp** [comment]
As the message says, we don't currently provide libc for Android. You need the NDK alongside Zig.

**Comment by isJDongYa** [comment]
> As the message says, we don't currently provide libc for Android. You need the NDK alongside Zig.

Yes, but i can't find any way to link libc.so at the android sdk path.

As the msg, I use lib.AddSystemLibrary, but it cant change to use the libc in android sdk.

When I use "lib.LinkSystemLibrary("abosolute path to libc.so")", the error say it can't  use absolute paht to link sys lib.

**Comment by tsokano** [comment]
Do you already know? This is a good reference.
https://github.com/silbinarywolf/zig-android-sdk/blob/main/src/androidbuild/tools.zig

**Comment by Epigeos-com** [comment]
Did anyone figure this out? Like preferably as a minimal guide instead of a big and complicated repo
